### ISSUE TITLE
Missing information about sentinel-2 bands used in pretrained ResNet model

### ISSUE THREAD

**Issue by bettasimousss** [microsoft/torchgeo#693]
### Issue

Hello,

In torchgeo.model the resnet50 model pretrained on Sentinel-2 data uses 10 channels , but I cannot find anywhere which of the 13 channels are used and in what order ? 

Can you clarify that in the documentation ?

**Comment by adamjstewart** [comment]
@calebrob6 do you know the answer to this? I thought our models were pre-trained on SEN12MS with band set "s2-all" but that also has 13 channels so I'm not sure where the 10 channels comes from.

**Comment by adamjstewart** [comment]
I think I figured this out. I believe our pre-trained models were actually trained on So2Sat. Specifically on the Sentinel-2 bands of So2Sat:
```python
    all_s2_band_names = (                                                                
        "B02",                                                                           
        "B03",                                                                           
        "B04",                                                                           
        "B05",                                                                           
        "B06",                                                                           
        "B07",                                                                           
        "B08",                                                                           
        "B08A",                                                                          
        "B11 SWIR",                                                                      
        "B12 SWIR",                                                                      
    )           
```
However, these bands were reordered to put the RGB bands first:
```
    # this reorders the bands to put S2 RGB first, then remainder of S2                  
    reindex_to_rgb_first = [2, 1, 0, 3, 4, 5, 6, 7, 8, 9]     
```
When using these pre-trained models, you'll likely also want to use the following normalizations:
```
    band_means = torch.tensor(                                                           
        [                                                                                
            0.12375696117681859,                                                         
            0.1092774636368323,                                                          
            0.1010855203267882,                                                          
            0.1142398616114001,                                                          
            0.1592656692023089,                                                          
            0.18147236008771792,                                                         
            0.1745740312291377,                                                          
            0.19501607349635292,                                                         
            0.15428468872076637,                                                         
            0.10905050699570007,                                                         
        ]                                                                                
    )                                                                                    
                                                                                         
    band_stds = torch.tensor(                                                            
        [                                                                                
            0.03958795985905458,                                                         
            0.047778262752410296,                                                        
            0.06636616706371974,                                                         
            0.06358874912497474,                                                         
            0.07744387147984592,                                                         
            0.09101635085921553,                                                         
            0.09218466562387101,                                                         
            0.10164581233948201,                                                         
            0.09991773043519253,                                                         
            0.08780632509122865,                                                         
        ]                                                                                
    )                             
```
So to use these in TorchGeo, you would either want to use our `So2SatDataModule` if you only care about So2Sat, or use something like the following for general Sentinel-2 data:
```python
bands=["B04", "B03", "B02", "B05", "B06", "B07", "B08", "B8A", "B11", "B12"]
transforms = Normalize(mean=band_means, std=band_stds)
ds = Sentinel2(bands=bands, transforms=transforms)
```
Obviously, the above is not documented and is clearly more complicated than it should be. We're currently searching for better supervised datasets or better self-supervised training methods that will allow us to train on all Sentinel-2 bands, in order. Once we expand our catalog of pre-trained models, we'll do a much better job of documenting this and making it as simple to use as possible.
